btrix

```diff
@@ -123,7 +127,7 @@ fi
 if [[ $1 = "reset" ]]; then
     
     echo "Current context: $CONTEXT"
-    echo "Resetting k8s cluster will delete the database. Are you sure you want to do this?"
+    echo "Resetting k8s cluster will delete the database. It's advised to wait until all crawls have finished before resetting to avoid crawljobs persisting. Are you sure you want to do this?"
```

Comment:
We can actually add a `kubectl delete -n crawlers cjs` and `kubectl delete -n crawlers pjs` to delete any crawl and profile jobs as first step.